bin/build.sh

```diff
@@ -43,7 +43,8 @@ TEMP_ARGS=" -D maven.test.skip=true "
 #cp ./target/*jar .
 
 # Dockerfile has the mvn commands
-time ${DOCKER} build --no-cache -t ${DOCKER_TAG} .
+#time ${DOCKER} build --no-cache -t ${DOCKER_TAG} .
+time ${DOCKER} build -t ${DOCKER_TAG} .
```

Comment:
any reason for removing the `--no-cache` tag?

Reply:
It was put in to require complete re-builds of the docker container.  It was taken out to improve build speed.